whois-api/src/main/java/net/ripe/db/whois/api/mail/dequeue/MessageDequeue.java

```diff
@@ -272,9 +274,19 @@ private void handleMessageInContext(final String messageId, final MimeMessage me
 
     private void handleUpdates(final MailMessage mailMessage, final UpdateContext updateContext) {
         final List<Update> updates = updatesParser.parse(updateContext, mailMessage.getContentWithCredentials());
+        addWarnIfPasswordExists(updateContext, updates);
```

Comment:
I thought the story says that it should be controlled via feature flag ? If feature flag is set to false return warning and if set to true return error with default sets to false?

Reply:
Put this change behind a feature flag so it can be enabled or disabled independently of Whois releases. If the flag is OFF then return a warning (make this the default). If the flag is ON then return an error.